src/django/api/models/facility/facility_list_item_temp.py

```diff
@@ -201,7 +201,7 @@ def copy(item):
         FacilityListItemTemp.objects.create(
             id=item.id,
             source=item.source,
-            facility_id=item.facility,
+            facility_id=item.facility_id,
```

Comment:
@mazursasha1990 Can you explain pls, why did you do this change? Will it affect functionality implemented in https://opensupplyhub.atlassian.net/browse/OSDEV-507 ?